### PR TITLE
SOC-2796 | Update Titles UI on Post Card and Post Details

### PR DESCRIPTION
**Pull request by mleczey**
## Links

* https://wikia-inc.atlassian.net/browse/SOC-2796

## Description

Update Titles UI on Post Card and Post Details

## Reviewers

@rafalkalinski @bkoval

### CODE REVIEW

File: front/main/app/styles/module/_discussion.scss [hunk 1]
```diff
@@ -311,8 +311,10 @@ body.discussions {
 
 	.post-title {
 		color: $discussion-post-title-color;
-		line-height: $line-height-smallest;
-		margin: 0;
+		font-size: $type-smaller;
+		font-weight: 500;
+		line-height: 1.25;
```
Comment (rafalkalinski) [inline front/main/app/styles/module/_discussion.scss]: units? and don't we have a variable for that?

File: front/main/app/styles/module/_discussion.scss [hunk 2]
```diff
@@ -311,8 +311,10 @@ body.discussions {
 
 	.post-title {
 		color: $discussion-post-title-color;
-		line-height: $line-height-smallest;
-		margin: 0;
+		font-size: $type-smaller;
+		font-weight: 500;
```
Comment (rafalkalinski) [inline front/main/app/styles/module/_discussion.scss]: don't we have variable for that?

File: front/main/app/styles/module/_discussion.scss [hunk 3]
```diff
@@ -641,7 +643,7 @@ body.discussions {
 			margin-bottom: $post-details-post-margin-bottom;
 
 			.post-title {
-				font-size: $type-smallest;
+				line-height: 1.25;
```
Comment (rafalkalinski) [inline front/main/app/styles/module/_discussion.scss]: units?

File: front/main/app/styles/module/_discussion.scss (hunk 2 above)
Comment (mleczey) [inline front/main/app/styles/module/_discussion.scss]: Unfortunately there is no variable for font weight 500.

**Comment by rafalkalinski** [comment]
LGTM

File: front/main/app/styles/module/_discussion.scss [hunk 4]
```diff
@@ -311,8 +311,10 @@ body.discussions {
 
 	.post-title {
 		color: $discussion-post-title-color;
-		line-height: $line-height-smallest;
-		margin: 0;
+		font-size: $type-smaller;
+		font-weight: 500;
+		line-height: $post-details-content-line-height;
+		margin: 0 0 5px 0;
```
Comment (bkoval) [inline front/main/app/styles/module/_discussion.scss]: 0 0 5px should be enough here

File: front/main/app/styles/module/_discussion.scss [hunk 5]
```diff
@@ -654,9 +652,9 @@ body.discussions {
 	&.post {
 
 		.post-detail .post-title {
-			font-size: $type-smaller;
-			line-height: $line-height-smaller;
-			margin-bottom: 18px;
+			font-size: $type-small;
+			line-height: $line-height-base;
+			margin: 7px 0 8px 0;
```
Comment (bkoval) [inline front/main/app/styles/module/_discussion.scss]: 7px 0 8px should be enough here

File: front/main/app/styles/module/_discussion.scss [hunk 6]
```diff
@@ -771,6 +769,14 @@ body.discussions {
 
 		.post-detail {
 			box-shadow: 0 1px 1px 0 rgba(0, 0, 0, 0.1);
+
+			.discussion-content {
+				.post-title {
+					font-size: $type-smaller;
+					line-height: $post-details-content-line-height;
+					margin: 5px 0 4px 0;
```
Comment (bkoval) [inline front/main/app/styles/module/_discussion.scss]: Same here, if you use 3 numbers for margin / padding, the middle one will be applied to both sides.

**Pull request merged**